packages/web3torrent/src/pages/file/File.tsx

```diff
@@ -61,8 +61,26 @@ const File: React.FC<RouteComponentProps> = () => {
           </FormButton>
           <div className="subtitle">
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-              incididunt ut labore et dolore magna aliqua.
+              <strong>How do I pay for the download?</strong>
+              <br />
+              When you click "Start Download", you'll be asked to allocate an amount of ETH so
+              Web3Torrent can collect payments on your behalf and transfer those funds to the file
+              owner. Unlike other systems, the payment is not upfront; instead, you pay as you
```

Comment:
I would replace __file owner__ with seeder, as:
- A Seeder of a file is not, necessarily the same as the file owner.
- Ownership is a pretty.... complicated word when we are talking about torrents. :-P